polkadot/node/network/statement-distribution/src/v2/mod.rs

```diff
@@ -542,6 +556,13 @@ pub(crate) async fn handle_active_leaves_update<Context>(
 		);
 	}
 
+	gum::debug!(
```

Comment:
```suggestion
	gum::trace!(
```

Reply:
just once per leaf update, not too spammy.